src/fondant/cli.py

```diff
@@ -335,6 +365,38 @@ def register_run(parent_parser):
 
     kubeflow_parser.set_defaults(func=run_kfp)
 
+    # Vertex runner parser
+    vertex_parser.add_argument(
+        "ref",
+        help="""Reference to the pipeline to run, can be a path to a spec file or
+            a module containing the pipeline instance that will be compiled first (e.g. pipeline.py)
+            """,
+        action="store",
+    )
+    vertex_parser.add_argument(
+        "--project-id",
+        help="""The project id of the GCP project used to submit the pipeline""",
+    )
+    vertex_parser.add_argument(
+        "--project-region",
+        help="The project region of the GCP project used to submit the pipeline ",
```

Comment:
This is on a per pipeline basis right? I tested it and it works. So can we update the help message to reflect hits? Now it seems like the region is tied to the project.

Reply:
Indeed this is the region where the pipeline is run and can be different from the project region. Updated the description